src/clusterfuzz/_internal/bot/tasks/cron/fuzz_strategy_selection.py

```diff
@@ -0,0 +1,212 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Fuzzing strategy selection cron job.
+
+Runs multi-armed bandit experiments for fuzzing strategy selection.
```

Comment:
Just curious: How much improvement does this bring in practice?

Reply:
We never got a chance to validate this :)